SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialActorChannel.h

```diff
@@ -346,4 +356,8 @@ class SPATIALGDK_API USpatialActorChannel : public UActorChannel
 	// before the actor holding the position for all the hierarchy, it can immediately attempt to migrate back.
 	// Using this timestamp, we can back off attempting migrations for a while.
 	uint64 AuthorityReceivedTimestamp;
+
+	// In case the actor's owner did not have an entity ID when trying to set interest to it
+	// We set this flag in order to try to add interest as soon as possible.
+	bool bNeedOwnerInterestUpdate;
```

Comment:
I did that because I did not see a straightforward way of checking if an interest update was needed. It would require accessing the owner, seeing that it is ready, and checking the interest component to see if the query is well setup.